ci-operator/config/openshift/microshift/openshift-microshift-main.yaml

```diff
@@ -1,21 +1,21 @@
 base_images:
   cli:
-    name: "4.10"
+    name: "4.11"
```

Comment:
We're using 4.10 code for our first delivery until the kube rebase is stable in OCP and we can update to 4.11/4.12. Will this change to 4.11 have any impact on the desire to stay on 4.10 for a few more weeks?